recipes/ppci/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "ppci" %}
+{% set version = "0.5.4" %}
+{% set   sha256 = "d5400cc043ca3eba0063f5dc4025747352fb03f950bd4b21e67bfa3f633966a3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
looks like you're missing run time dependencies. Error message for import failure for `colorama`. There may be others, but I'll leave it to you to find the dependencies.

Reply:
Okay, I added the colorama dependency.